### PR TITLE
Check drug availability in prescription

### PR DESCRIPTION
**Pull request by Guimino**
Check the existence of stock of the medicine to be prescribed

### CODE REVIEW

**Comment by Guimino** [comment]
[Check stock availability of Drug](https://trello.com/c/7z3tasHY/115-check-stock-availability-of-drug)

**Comment by ynurmahomed** [comment]
[8.1.1. Develop component to provide information about the availability of a drug when the clinician  places prescription order](https://trello.com/c/JoatAvRk/46-811-develop-component-to-provide-information-about-the-availability-of-a-drug-when-the-clinician-places-prescription-order)

File: app/common/prescription/directives/prescription.js [hunk 1]
```diff
@@ -679,6 +681,13 @@
       return angular.element('#addProviderAndPrescriptionDateModel');
     }
 
+    function verifyDrugAvailability(drug){
+      drugService.getDrugStock(drug,sessionService.getCurrentLocation()).then(function(data){
+        if(_.isEmpty(data.results)){
+         notifier.warning($filter('translate')('COMMON_MESSAGE_DRUG_WITHOUT_STOCK_AVAILABILITY'));;
+        }
+      });
```
Comment (ynurmahomed) [inline app/common/prescription/directives/prescription.js]: When this call fails the user should be notified that something went wrong. Should call ```.catch()``` here.

Comment (Guimino) [reply app/common/prescription/directives/prescription.js]: there was a declared attribute in Batch resource that no longer existed in the domain class  that was causing this error.  Fixes by https://github.com/esaude/openmrs-module-inventory.poc/pull/7

**Pull request merged**